flink-runtime/src/test/java/org/apache/flink/runtime/jobmanager/JobManagerProcessUtilsTest.java

```diff
@@ -37,23 +38,21 @@
 
 import static org.apache.flink.runtime.jobmanager.JobManagerProcessUtils.JM_LEGACY_HEAP_OPTIONS;
 import static org.apache.flink.runtime.jobmanager.JobManagerProcessUtils.JM_PROCESS_MEMORY_OPTIONS;
-import static org.hamcrest.CoreMatchers.containsString;
-import static org.hamcrest.CoreMatchers.hasItem;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.collection.IsArrayWithSize.arrayWithSize;
-import static org.hamcrest.collection.IsMapContaining.hasKey;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 
 /** Tests for {@link JobManagerProcessUtils}. */
-public class JobManagerProcessUtilsTest extends ProcessMemoryUtilsTestBase<JobManagerProcessSpec> {
+class JobManagerProcessUtilsTest extends ProcessMemoryUtilsTestBase<JobManagerProcessSpec> {
```

Comment:
Yeah the class in jobmanager module. 

I migrated it because `JobManagerProcessUtilsTest` and `TaskExecutorProcessUtilsTest` extends `ProcessMemoryUtilsTestBase`.

I want to migrate these 3 classes to junit5 together, however `ProcessMemoryUtilsTestBase` is migrated at https://github.com/apache/flink/pull/23199, so I updated `JobManagerProcessUtilsTest` and `TaskExecutorProcessUtilsTest` at this PR.

Do you think it is ok?

Reply:
It's ok to me.